packs/feats/you-dont-smell-right.json

```diff
@@ -28,7 +28,33 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Werecreature.YouDontSmellRight.Label",
+                "option": "seek-disguise",
+                "toggleable": true
+            },
+            {
+                "acuity": "imprecise",
+                "key": "Sense",
+                "predicate": [
+                    "feat:feral-senses"
+                ],
+                "range": 60,
+                "selector": "scent"
+            },
```

Comment:
This should be an unpredicated 30, and a predicated 60 on having the feat and being in hybrid or animal shape (when feral senses would be active)